docs/template/public/less/_common.less

```diff
@@ -0,0 +1,147 @@
+// Common styles
+
+@import '_variables';
+
+body.kss-body {
+    background: @white;
+    overflow: visible;
+    font-size: 15px;
+    font-family: 'Open Sans', sans-serif;
+    padding: 0;
+    max-width: 100%;
+    padding-top: 100px;
+    padding-right:0!important;
+}
+
+#kss-wrapper {
+    height: auto;
+}
+
+.active.demo-container {
+    background-color: @screen-bg_gd;
+    padding: 15px;
+}
+
+#kss-header h1 {
+    font-size: 1.5em;
+    margin-top: 5/3em;
+    margin-left: 4/3em;
+}
+
+h1.kss-title {
+    color: @blue-heading;
+    margin-bottom: 30px;
+    font-weight: 500;
+    font-family: 'Dosis', sans-serif;
+}
+
+#kss-main {
+    padding-left: 15px;
+    padding-bottom: 40px;
+
+    &.col-md-12{
+        max-width: 1000px;
+    }
+
+}
+
+.kss-mod-list {
+    width: 100%;
+}
+
+.kss-mod-desc {
+    display: inline-block;
+    width: 69%;
+    padding: 10px 0;
+}
+
+.kss-mod-head {
+    border-top: 1px solid @grey-border;
+    padding: 20px 0 20px;
+    margin-top: 20px;
+}
+
+.kss-mod-name {
+    width: 30%;
+    display: inline-block;
+    text-align: right;
```

Comment:
lets use the bootstrap text-right instead of defining text align in the stylesheet